css-four-column-layout/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Barlow:wght@400;500;600;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="container padding-lr">
+    <div class="row">
+      <div class="column-full">
+        <h2 class="text-center">How it works</h2>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-one-fourth">
+        <div class="row justify-align-center">
+          <img src="images/screen-1.png" alt="screen1" class="phone-size">
+        </div>
+      </div>
+      <div class="column-one-fourth">
+        <div class="row justify-align-center">
+          <img src="images/screen-2.png" alt="screen2" class="phone-size">
+        </div>
+      </div>
+      <div class="column-one-fourth">
+        <div class="row justify-align-center">
+          <img src="images/screen-3.png" alt="screen3" class="phone-size">
+        </div>
+      </div>
+      <div class="column-one-fourth">
+        <div class="row justify-align-center">
+          <img src="images/screen-4.png" alt="screen4" class="phone-size">
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-one-fourth">
+        <div class="text-center">
+          <img src="images/switch-icon.png" alt="switch icon" class="icon-size">
+          <h4>App switching</h4>
+          <p>Toggles all your apps on and off.</p>
+        </div>
+      </div>
+      <div class="column-one-fourth">
+        <div class="text-center">
+          <img src="images/filter-icon.png" alt="filter icon" class="icon-size">
+          <h4>Trip Filtering</h4>
+          <p>Evaluates your trip requests</p>
+        </div>
+      </div>
+      <div class="column-one-fourth">
+        <div class="text-center">
+          <img src="images/slider-icon.png" alt="slider icon" class="icon-size">
+          <h4>Service Filters</h4>
+          <p>Accepts only your ideal trips.</p>
+        </div>
+      </div>
+      <div class="column-one-fourth">
+        <div class="text-center">
+          <img src="images/bars-icon.png" alt="bars icon" class="icon-size">
+          <h4>Stats + Mileage</h4>
+          <p>Tracks your trips across all apps.</p>
+        </div>
+      </div>
+    </div>
+  </div>
```

Comment:
I can see why you would separate the images out into their own row from a layout class perspective, but this makes it seem like the images are unrelated to the content that follows it when reading the HTML. Instead, I would make four `column-one-fourth` elements, each of which contains the image, icon, subheader, and descriptive text.